### PR TITLE
Physical processor count fix

### PR DESCRIPTION
**Pull request by ellesummer**
Attempted to address bug in physical processor count for linux and unix. Still needs to be tested.

### CODE REVIEW

File: oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java [hunk 1]
```diff
@@ -142,14 +142,15 @@ private void initVars() {
...
         logProcs.add(new LogicalProcessor(currentProcessor, currentCore, currentPackage,
                 numaNodeMap.getOrDefault(currentProcessor, 0)));
-        Set<Integer> physProcs = new HashSet<>();
+        Set<String> physProcPkgs = new HashSet<>();
         Set<Integer> physPkgs = new HashSet<>();
         for (LogicalProcessor logProc : logProcs) {
-            physProcs.add(logProc.getPhysicalProcessorNumber());
-            physPkgs.add(logProc.getPhysicalPackageNumber());
+            int pkg = logProc.getPhysicalPackageNumber();
+            physProcPkgs.add("" + logProc.getPhysicalProcessorNumber() + ":" + pkg);
```
[the remark this answers is not in the document]
Comment (ellesummer) [reply oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java]: Ah of course. I had the "" in the beginning before I forgot to include the delimiter. I'll fix it now.

**Comment by dbwiddis** [comment]
Merging. Something's up with the Travis JDK 11 script so that test indicates failing, but it's not an issue.

**Comment by dbwiddis** [comment]
Thanks for your contribution to Open Source!

**Comment by ellesummer** [comment]
Thanks for the guidance!

**Comment by dbwiddis** [comment]
I've updated the CONTRIBUTING.md doc based on your feedback, let me know if it's more clear now!

**Comment by ellesummer** [comment]
Much clearer, thanks.

**Pull request merged**